backend/backend/settings/deps/restframework.py

```diff
@@ -24,5 +24,7 @@
 
 SIMPLE_JWT = {
     'ACCESS_TOKEN_LIFETIME': timedelta(minutes=int(os.environ.get('ACCESS_TOKEN_LIFETIME', 24*60))),
+    'REFRESH_TOKEN_LIFETIME': timedelta(minutes=int(os.environ.get('REFRESH_TOKEN_LIFETIME', 24*60*7))),
+    'ROTATE_REFRESH_TOKENS': True,
```

Comment:
Can you add the link to the documentation ?
Here there a way to do both ?

Reply:
Here is the documentation I was mentioning https://github.com/davesque/django-rest-framework-simplejwt#settings 

I re-read it and I may have misunderstood it: the issue I have described may occur only if the BLACKLIST_AFTER_ROTATION option is on (and it's not at the moment). Otherwise, the old token isn't invalidated, it'll will just expire by itself. I'll have to test it to be sure.